cms/jinja2/templates/components/contact_details/contact_details.html

```diff
@@ -5,17 +5,34 @@
         <h2 class="ons-u-fs-l">{{ _("Contact details") }}</h2>
 
         {% call onsTextIndent() %}
-            <p class="contact-details__line">
-                <strong>Name</strong>: {{ contact_details.name }}
-            </p>
-            <p class="contact-details__line">
-                <strong>Email</strong>: <a href="mailto:{{ contact_details.email }}">{{ contact_details.email }}</a>
-            </p>
-            {% if contact_details.phone %}
-                <p class="contact-details__line">
-                    <strong>Telephone</strong>: <a href="tel:{{ contact_details.phone|urlencode() }}">{{ contact_details.phone }}</a>
-                </p>
-            {% endif %}
+            <dl>
```

Comment:
Is there a DS component that we could use?

Reply:
👍  you're right we do, I've replaced with the description list component, the styling's a little different because it uses a grid instead, but I think that's okay.